react/basic-example/package.json

```diff
@@ -7,13 +7,19 @@
     "talkjs": "^0.3.0"
   },
   "devDependencies": {
-    "react-scripts": "1.0.13"
+    "react-scripts": "^2.1.8"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "browserslist": [
+    ">0.2%",
+    "not dead",
+    "not ie <= 11",
+    "not op_mini all"
+  ]
```

Comment:
what's this?

Reply:
This specifies the browser that the app should automatically open in (or rather, which one's it shouldn't open in) - npm added this.

Reply:
ok.